hash_ring_configuration.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
shouldn't this file be in the `ring` package as well, since the `ReplicaPoints` are part of the ring implementation as well.